index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/footer.css">
+    <link rel="stylesheet" href="css/body.css">
+    <link rel="stylesheet" href="css/all.css">
+    <link rel="stylesheet" href="css/top.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Apple</title>
+</head>
+<body>
+    <div class=container>
+    <div class="top">
+<nav>
+    <ul>
+        <li><a href=""><div class="button apple"><i class="fa fa-apple "></i></div></a></li>
+        <li><a href=""><div class="button">Store</div></a></li>
+        <li><a href=""><div class="button">Mac</div></a></li>
+        <li><a href=""><div class="button">iPod</div></a></li>
+        <li><a href=""><div class="button">iPhone</div></a></li>
+        <li><a href=""><div class="button">iPad</div></a></li>
+        <li><a href=""><div class="button">iTunes</div></a></li>
+        <li><a href=""><div class="button">Support</div></a></li>
+        <li><div class="button last">
+            <div class="laster">
+            <div class="search"><i class="fa fa-search"></i></div>
+            <input type="text">
+        </div>
+    </div>
+        </li>
+    </ul>
+</nav>
+<div>
+    <a href="">
+        <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_title_small.png" width="980" height="104" alt="What will your verse be?">
+</a>
+</div>
+<div>
+    <a href="">
+        <img class="second" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_subhead_small.png" width="980" height="82" alt="Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story.">
+    </a>
+</div>
+<div>
+                <a href="" class="bottom-title">Watch the “Your Verse” film and explore the stories</a>
+</div>
+       </div>
+    </div>
+    <div class="mid-section">
+        <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5c.jpg" width="360" height="200" alt="iPhone 5c. For the colorful.">
```

Comment:
All Images should be wrapped in `a` tags thanks

Reply:
Thanks... I have updated that. Please do take a look again.

Reply:
Also,can i please get your full name so that i can pick your name when filling the project completion form. Thanks.

Reply:
Also,my Read.me file is in my master branch.